Gemfile

```diff
@@ -2,12 +2,15 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gem 'rails',      '6.0.0'
+gem 'sqlite3'
```

Comment:
Shouldn't we setup devise to use postgres?

Reply:
Mooj is working on that right now